templates/pkg/crd_manager.go.tpl

```diff
@@ -0,0 +1,130 @@
+{{ template "boilerplate" }}
```

Comment:
nit: adding a comment - similar to what `controller-gen` does - to the boilerplate saying `// Code generated by ack-generate. DO NOT EDIT.`  might be useful

Reply:
ah, yes great point. lemme update.